README.md

```diff
@@ -1,3 +1,61 @@
+| -- | -- | -- | -- |
+| 1 | Text preprocessing and similarity | [lecture](lectures/M1-text_preprocessing_and_similarity.pdf) | [exercises](exercises/E1) |
+| 2 | Text classification and similarity | [lecture](lectures/M2-text_classification_and_clustering.pdf) | [exercises](exercises/E2) |
+| 3 | Search engine architecture and basic retrieval models | [lecture](lectures/M3-se_architecture_and_retrieval_models.pdf) | [exercises](exercises/E3) |
+| 4 | Advanced retrieval models | [lecture](lectures/M4-advanced_retrieval_models.pdf) | [exercises](exercises/E4) |
+| 5 | Retrieval evaluation | [lecture](lectures/M5-retrieval_evaluation.pdf) | [exercises](exercises/E5) |
+| 6 | Knowledge bases and entity retrieval | [lecture](lectures/M6-knowledge_bases_and_entity_retrieval.pdf) | [exercises](exercises/E6) |
+| 7 | Entity linking | [lecture](lectures/M7-entity_linking.pdf) | [exercises](exercises/E7) |
+| 8 | Semantic search | [lecture](lectures/M8-eos.pdf) | [exercises](exercises/E8) |
+| 9 | Word embeddings and dense retrieval | [lecture](lectures/M9-neural-ir-1.pdf) | [exercises](exercises/E9) |
+| 10 | Transformer-based models | [lecture](lectures/M10-neural-ir-2.pdf) | [exercises](exercises/E10) |
+| 11 | Conversational Information Access | [lecture](lectures/M11-CIA.pdf) |  |
+| 12 | Advanced evaluation | [lecture](lectures/M12-advanced_evaluation.pdf) |  |
+| 13 | Conversational Search Systems | [lecture](lectures/M13-css.pdf) | [exercises](exercises/E13) |
+| 14 | Fairness and transparency in IR | [lecture](lectures/M14-fairness_transparency_ir.pdf) | [exercises](exercises/E14) |
+
+## Assignments
+
+During the course, students had to complete 9 assignments (deliverables) individually. These were graded and accounted for 50% of the project work (i.e., 20% of the final grade). Students were given a certain time limit (typically 10 days) to complete each assignment. For each assignment students were provided with the Python file with code skeleton which they were expected to complete according to the task description. Assignments were graded automatically, using a combination of public and hidden tests. Public tests were released together with the assignment; if the solution passed these tests, it was likely correct. However, in addition to those tests, there were some hidden tests that the solution was tested against after submission; these typically contained larger inputs/datasets, corner cases, or other inputs in order to test that the student fully understood the methods and/or followed the instructions.
+
+| **Id** | **Topic** | **Points** |
+| -- | -- | -- |
+| 1 | Classifier: Feature extraction | 2 |
+| 2 | Classifier: Evaluation | 3 |
+| 3 | Classifier: Training and evaluation | 5 |
+| 4 | Search engine: Indexing | 4 |
+| 5 | Search engine: Scoring | 7 |
+| 6 | Search engine: Evaluation | 4 |
+| 7 | Entity retrieval | 8 |
+| 8 | Entity ranking | 7 |
+| 9 | Learning to Rank | 10 |
```

Comment:
Use the labels used in Canvas: A1.1, A1.2, ... A3.3

Reply:
Done